lib/sycamore/sycamore/tests/integration/connectors/opensearch/test_pdf_to_opensearch.py

```diff
@@ -133,12 +138,7 @@ def test_pdf_to_opensearch_with_llm_caching():
                 )
             )
         )
-
-        ds.write.opensearch(
-            os_client_args=os_client_args,
-            index_name="toyindex",
-            index_settings=index_settings,
-        )
+        ds.write.opensearch()
```

Comment:
I tend to think that one is more likely to read from and write to different indices, so maybe the index name is still a parameter to the read.opensearch() and write.opensearch() calls, but the OS config is in the config object.

Reply:
They're all still parameters, they just default to reading from the config if not specified. You can still do ds.write.opensearch(index="something")

Reply:
we have a couple notebooks I believe that are using the same index in a bunch of places

Reply:
write, then query etc